brain_games/game.py

```diff
@@ -0,0 +1,21 @@
+from prompt import string
+
+
+def game(name, func):
+    count = 0
+    while count < 3:
+        task_params = func()
+        print(task_params["task"])
+        print(f"Question: {task_params['expression']}")
+        correct_answer = task_params["correct answer"]
+        answer = string("Your answer: ")
+        if correct_answer == answer:
+            print("Correct!")
+            count += 1
+        else:
+            print(
```

Comment:
this print is repeated in both games, let's make a function for them in `cli` module